package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "trezor-suite",
-    "version": "1.0.0",
+    "version": "05.13.2020.BETA",
```

Comment:
do we really want to use different syntax than in suite-data ( "version": "05.13.2020-beta"),

Reply:
just a nitpick but major version shouldn't with 05? maybe it should be: `0.0.51320` instead?

Reply:
or even better `0.0.20200513` this way we'll have not retarded date format with possible valid incrementation